bitvision/bitvision.py

```diff
@@ -8,9 +8,10 @@
 import gdax
 import urwid
 
-from engine.data_bus import Dataset, Vector
-from engine.transformers import Transformer
-from engine.model import Model
+sys.path.insert(0, "engine")
```

Comment:
So I recently learned about a better way to do this. We should turn the `engine/` directory into a Python package by adding an `__init__.py` file, and then expose these objects/functions in that file by adding:
```python
from .data_bus import Dataset, Fetch
from .transformers import Transformer
from .model import Model
```
This way, if we want to use those objects/functions in `bitvision.py`, all we have to do is:
```python
from engine import Dataset, Fetch, Transformer, Model
```

Reply:
That's clutch

Reply:
I'm not sure if I got this to work properly. PyCharm keeps giving me errors but it seems alright in the console, idk

Reply:
Got it.